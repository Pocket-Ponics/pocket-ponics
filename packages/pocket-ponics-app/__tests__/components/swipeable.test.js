import React from 'react'
import { shallow } from 'enzyme'

import Swipeable from '../../components/greenhouse/swipeable'

describe('Stats Display', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<Swipeable 
				greenhouse={{ tiers: [null, null, null, null], history: [] }} 
				navigation={{}}/>
		).html()
		expect(tree).toMatchSnapshot()
	})
})