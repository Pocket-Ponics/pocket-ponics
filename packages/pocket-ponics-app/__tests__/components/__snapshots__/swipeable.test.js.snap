// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats Display renders correctly 1`] = `"<RCTScrollView style=\\"width:750px;background-color:#472600\\"><View><View style=\\"height:600px;align-items:center\\"><Text style=\\"color:#FFFFFF;font-size:40px\\"></Text><View style=\\"color:#472600;align-items:center;justify-content:center;margin:10px;flex:1\\"><View style=\\"flex:1.8;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:1.1904761904761905px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View><View style=\\"flex:1;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:2.142857142857143px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View><View style=\\"flex:1;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:2.142857142857143px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View><View style=\\"flex:1;width:100%;align-items:center;justify-content:center;opacity:1\\"><View style=\\"flex:1;flex-direction:row;justify-content:center;aspect-ratio:2.142857142857143px\\"><Image source=\\"1\\" style=\\"0:[object Object];1:[object Object];2:[object Object]\\"></Image></View></View></View></View><View style=\\"margin-top:20px\\"><Text style=\\"font-size:40px;color:#FFFFFF;text-align:center\\">Current Stats</Text><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Battery: %</Text></View><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Water: %</Text></View><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Nutrients:  %</Text></View></View><View style=\\"align-items:center;justify-content:center;flex:1;margin-top:20px\\"><Text style=\\"color:#FFFFFF;font-size:40px\\">Historical Data</Text><View><RNSVGSvgView height=\\"287.5\\" width=\\"750\\" bbWidth=\\"750\\" bbHeight=\\"287.5\\" style=\\"0:[object Object];2:[object Object];3:[object Object]\\" focusable=\\"false\\"><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGRect matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"1,backgroundGradient\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" width=\\"100%\\" height=\\"287.5\\" rx=\\"0\\" ry=\\"0\\"></RNSVGRect><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGRect matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"150.5\\" y=\\"10.75\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4292163463\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" width=\\"16\\" height=\\"16\\" rx=\\"8\\" ry=\\"8\\"></RNSVGRect><RNSVGText matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"170.5\\" y=\\"24.375\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4294967295\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"><RNSVGTSpan matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"\\" y=\\"\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" content=\\"Battery\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"></RNSVGTSpan></RNSVGText></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGRect matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"344\\" y=\\"10.75\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4284065483\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" width=\\"16\\" height=\\"16\\" rx=\\"8\\" ry=\\"8\\"></RNSVGRect><RNSVGText matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"364\\" y=\\"24.375\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4294967295\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"><RNSVGTSpan matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"\\" y=\\"\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" content=\\"Water\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"></RNSVGTSpan></RNSVGText></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGRect matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"522.5\\" y=\\"10.75\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4286884721\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" width=\\"16\\" height=\\"16\\" rx=\\"8\\" ry=\\"8\\"></RNSVGRect><RNSVGText matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"542.5\\" y=\\"24.375\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4294967295\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"><RNSVGTSpan matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"\\" y=\\"\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" content=\\"Nutrient\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"></RNSVGTSpan></RNSVGText></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,37.5\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"37.5\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGDefs><RNSVGLinearGradient x1=\\"0\\" y1=\\"250\\" x2=\\"750\\" y2=\\"0\\" name=\\"backgroundGradient\\" gradient=\\"0,-12114432,1,-12114432\\" gradientUnits=\\"0\\" gradientTransform=\\"1,0,0,1,0,0\\"></RNSVGLinearGradient><RNSVGLinearGradient x1=\\"0\\" y1=\\"0\\" x2=\\"0\\" y2=\\"250\\" name=\\"fillShadowGradient\\" gradient=\\"0,452984831,1,16777215\\" gradientUnits=\\"0\\" gradientTransform=\\"1,0,0,1,0,0\\"></RNSVGLinearGradient></RNSVGDefs><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGLine matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"stroke,strokeWidth,strokeDasharray\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke=\\"0,872415231\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-dasharray=\\"5,10\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" x1=\\"64\\" y1=\\"16\\" x2=\\"750\\" y2=\\"16\\"></RNSVGLine><RNSVGLine matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"stroke,strokeWidth,strokeDasharray\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke=\\"0,872415231\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-dasharray=\\"5,10\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" x1=\\"64\\" y1=\\"78.5\\" x2=\\"750\\" y2=\\"78.5\\"></RNSVGLine><RNSVGLine matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"stroke,strokeWidth,strokeDasharray\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke=\\"0,872415231\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-dasharray=\\"5,10\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" x1=\\"64\\" y1=\\"141\\" x2=\\"750\\" y2=\\"141\\"></RNSVGLine><RNSVGLine matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"stroke,strokeWidth,strokeDasharray\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke=\\"0,872415231\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-dasharray=\\"5,10\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" x1=\\"64\\" y1=\\"203.5\\" x2=\\"750\\" y2=\\"203.5\\"></RNSVGLine></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGText matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"52\\" originY=\\"199.5\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"52\\" y=\\"199.5\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4294967295\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"><RNSVGTSpan matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"\\" y=\\"\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" content=\\"NaN%\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"></RNSVGTSpan></RNSVGText><RNSVGText matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"52\\" originY=\\"141\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"52\\" y=\\"141\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4294967295\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"><RNSVGTSpan matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"\\" y=\\"\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" content=\\"NaN%\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"></RNSVGTSpan></RNSVGText><RNSVGText matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"52\\" originY=\\"82.5\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"52\\" y=\\"82.5\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4294967295\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"><RNSVGTSpan matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"\\" y=\\"\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" content=\\"NaN%\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"></RNSVGTSpan></RNSVGText><RNSVGText matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"52\\" originY=\\"24\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"52\\" y=\\"24\\" propList=\\"fill\\" opacity=\\"1\\" fill=\\"0,4294967295\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"><RNSVGTSpan matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"\\" y=\\"\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" content=\\"NaN%\\" font=\\"[object Object]\\" dx=\\"\\" dy=\\"\\" rotate=\\"\\"></RNSVGTSpan></RNSVGText></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"><RNSVGPath matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"fill,stroke,strokeWidth\\" opacity=\\"1\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke=\\"0,4292163463\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"2\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" d=\\"\\"></RNSVGPath><RNSVGPath matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"fill,stroke,strokeWidth\\" opacity=\\"1\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke=\\"0,4284065483\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"2\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" d=\\"\\"></RNSVGPath><RNSVGPath matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"fill,stroke,strokeWidth\\" opacity=\\"1\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke=\\"0,4286884721\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"2\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" d=\\"\\"></RNSVGPath></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"></RNSVGGroup><RNSVGGroup matrix=\\"1,0,0,1,0,0\\" rotation=\\"0\\" originX=\\"0\\" originY=\\"0\\" scaleX=\\"1\\" scaleY=\\"1\\" skewX=\\"0\\" skewY=\\"0\\" x=\\"0\\" y=\\"0\\" propList=\\"\\" opacity=\\"1\\" fill=\\"0,4278190080\\" fill-rule=\\"1\\" fill-opacity=\\"1\\" stroke-opacity=\\"1\\" stroke-linecap=\\"0\\" stroke-linejoin=\\"0\\" stroke-width=\\"1\\" stroke-miterlimit=\\"4\\" vector-effect=\\"0\\" font=\\"[object Object]\\"></RNSVGGroup></RNSVGGroup></RNSVGGroup></RNSVGSvgView></View></View></View></RCTScrollView>"`;
